api/SVGGeometryElement.json

```diff
@@ -0,0 +1,344 @@
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "getTotalLength": {
+        "__compat": {
+          "description": "getTotalLength()",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGeometryElement/getTotalLength",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
+            },
+            "firefox_android": {
+              "version_added": "53"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "getPointAtLength": {
+        "__compat": {
+          "description": "getPointAtLength()",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGeometryElement/getPointAtLength",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```

Comment:
Again, from the note this should be fixed for version 61 - please add note to 53 and reflect support in 61 - something like:

```
"firefox": [
  {
    "version_added": "53",
    "notes": "This property is currently only supported on <code><path></code> elements (where it was already supported in earlier Firefox versions through the <code>SVGPathElement</code> interface). Support for other elements will be added in <a href='...'>bug 1325320</a>."
  },
  {
    "version_added": "61"
  }
]
```